extension/pages/settings/settings.js

```diff
@@ -392,6 +392,111 @@ async function setupPreferences() {
 	fillSettings();
 	storageListeners.settings.push(updateSettings);
 
+	/* Search */
+	(() => {

```

Comment:
Why put it in a self-evaluating anonymous function? A better option would be a named function.